deliverystream/aws-kinesisfirehose-deliverystream.json

```diff
@@ -755,6 +807,72 @@
           "type": "boolean"
         }
       }
+    },
+    "HttpEndpointRequestConfiguration": {
+      "type": "object",
+      "additionalProperties": false,
+      "properties": {
+        "ContentEncoding": {
+          "type": "string",
+          "enum": [
+            "NONE",
+            "GZIP"
+          ]
+        },
+        "CommonAttributes": {
+          "type": "array",
+          "uniqueItems": true,
+          "items": {
+            "$ref": "#/definitions/HttpEndpointCommonAttribute"
+          },
+          "minItems": 0,
+          "maxItems": 50
+        }
+      }
+    },
+    "HttpEndpointCommonAttribute": {
+      "type": "object",
+      "additionalProperties": false,
+      "properties": {
+        "AttributeName": {
+          "type": "string",
+          "minLength": 1,
+          "maxLength": 256
+        },
+        "AttributeValue": {
+          "type": "string",
+          "minLength": 0,
+          "maxLength": 1024
+        }
+      },
+      "required": [
+        "AttributeName",
+        "AttributeValue"
+      ]
+    },
+    "HttpEndpointConfiguration": {
+      "type": "object",
+      "additionalProperties": false,
+      "properties": {
+        "Url": {
+          "type": "string",
+          "minLength": 1,
+          "maxLength": 1000
+        },
+        "AccessKey": {
+          "type": "string",
+          "minLength": 0,
+          "maxLength": 4096
+        },
+        "Name": {
+          "type": "string",
+          "minLength": 1,
+          "maxLength": 256
+        }
+      },
+      "required": [
+        "Url"
```

Comment:
Does it make sense to add pattern here. 
https://docs.aws.amazon.com/firehose/latest/APIReference/API_HttpEndpointConfiguration.html

Reply:
Will be good to add pattern here, so invalid template can be rejected while validation

Reply:
Not sure if we are talking only about the URL but wanted to call out if it makes sense to specify pattern for other fields (Name & AccessKey) as well?